benchmark_runner/common/ocp_resources/create_lso.py

```diff
@@ -29,7 +31,15 @@ def create_lso(self, upgrade_version: str):
         else:
             for resource in self.__resource_list:
                 logger.info(f'run {resource}')
-                self.__oc.create_async(yaml=os.path.join(self.__path, resource))
+                if resource.endswith('.sh'):
+                    if '01_delete_disks.sh' == resource and self.__lso_disk_id:
+                        disk = f'/dev/disk/by-id/{self.__lso_disk_id}'
+                        delete_node_disk = f""" podman run --authfile /var/lib/kubelet/config.json --rm --privileged --device "$(readlink -e {disk})" --entrypoint ceph-bluestore-tool quay.io/ceph/ceph:v19 zap-device --dev "$(readlink -e {disk})" --yes-i-really-really-mean-it """
```

Comment:
It really should be
```
"$(readlink -e "{disk}")"
```
I doubt you're going to find spaces or tabs in any disk name, but it's a good idea to be defensive about this.

Reply:
The '' there is only for the Python string.  When you pass it to the shell, you need to quote it too.